src/ui.rs

```diff
@@ -120,7 +154,17 @@ pub fn event_action(state: &mut State, event: &Event) -> Action {
                 state.right_shift = true;
                 Action::Noop
             }
-            _ => Action::Noop,
+            _ => {
+                // convert scancode to keycode via virtual mapping
+                let keycode = match sdl2::keyboard::Keycode::from_scancode(*scan) {
```

Comment:
Added virtual mapping handled here, but this doesn't actually work on my end, the keycode produced is "semicolon"

So now there is handling for semicolon, colon, and virtual mapping
as mentioned [here](https://wiki.libsdl.org/SDL_Keycode)

Reply:
What about plain old colon as keycode https://rust-sdl2.github.io/rust-sdl2/sdl2/keyboard/enum.Keycode.html#variant.Colon . Def doesn't work?

Reply:
Yeah, that doesn't work on my end nor does `KpColon`...

Reply:
text input does seem like an ugly hack, but it might be the best bet. FYI, I was too lazy to handle this earlier, which is why I went with H for help instead of question mark like I originally planned. So thanks for taking this on.

Reply:
Okay, I'm on it

Reply:
Hey, and seeing as you have an open pull request, can you please put the stuff about sorting below the stuff about basic controls in the README :)